aws/data_source_aws_iam_session_context.go

```diff
@@ -0,0 +1,135 @@
+package aws
+
+import (
+	"fmt"
+	"regexp"
+	"strings"
+
+	"github.com/aws/aws-sdk-go/aws/arn"
+	"github.com/aws/aws-sdk-go/service/iam"
+	"github.com/hashicorp/aws-sdk-go-base/tfawserr"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/terraform-providers/terraform-provider-aws/aws/internal/service/iam/finder"
+	"github.com/terraform-providers/terraform-provider-aws/aws/internal/service/iam/waiter"
+	"github.com/terraform-providers/terraform-provider-aws/aws/internal/tfresource"
+)
+
+func dataSourceAwsIAMSessionContext() *schema.Resource {
+	return &schema.Resource{
+		Read: dataSourceAwsIAMSessionContextRead,
+
+		Schema: map[string]*schema.Schema{
+			"arn": {
+				Type:         schema.TypeString,
+				Required:     true,
+				ValidateFunc: validateArn,
+			},
+			"role_name": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"source_arn": {
```

Comment:
Swap with `session_name` for alphabetic ordering?

Reply:
it's now called `issuer_name` and should be in the correct order